build.psm1

```diff
@@ -1265,7 +1265,8 @@ function Start-PSBootstrap {
             if ($Package) {
                 try {
                     # We cannot guess if the user wants to run gem install as root
-                    Start-NativeExecution { gem install fpm ronn }
+                    Start-NativeExecution { gem install fpm -v 1.8.1 }
```

Comment:
We can do that later, please open an issue for it. For now, we need to unblock our release 😄